src/routes/root.ts

```diff
@@ -1,4 +1,4 @@
-import { Router } from 'express'
```

Comment:
Our convention is not to use default exports but instead import the explicit names of methods if possible. This ensures consistent naming throughout the application

Reply:
I align with the notion against default exports. However, what's your suggestion for this case where 'express-promise-router' exposes a single factory for the router instances as it's default?

Reply:
If that’s the case then we have no choice, then no worries